disco/extensions/upgrade_simulation/upgrades/automated_thermal_upgrades.py

```diff
@@ -23,36 +23,37 @@
 def determine_thermal_upgrades(
     job_name,
     master_path,
-    enable_pydss_solve,
-    pydss_volt_var_model,
     thermal_config,
+    upgrade_simulation_params_config,
     internal_upgrades_technical_catalog_filepath,
     thermal_upgrades_dss_filepath,
     upgraded_master_dss_filepath,
     output_json_thermal_upgrades_filepath,
     feeder_stats_json_file,
     thermal_upgrades_directory,
     overall_output_summary_filepath,
-    dc_ac_ratio,
     ignore_switch=True,
     verbose=False
 ):
+    if verbose:
+        logging.level = logging.DEBUG
     start_time = time.time()
     logger.info( f"Simulation start time: {start_time}")   
-    initial_simulation_params = {"enable_pydss_solve": enable_pydss_solve, "pydss_volt_var_model": pydss_volt_var_model,
-                                 "dc_ac_ratio": dc_ac_ratio}
-    logger.info("Initial simulation parameters: %s", initial_simulation_params)
+    analysis_params = SimulationParams(timepoint_multipliers=upgrade_simulation_params_config["timepoint_multipliers"], 
+                                       timeseries_analysis=upgrade_simulation_params_config["timeseries_analysis"])
+    # TODO analysis params validator is not working

```

Comment:
I'm good with merging after we know what is happening with this.

Reply:
Fixed after creating this as an UpgradeParamsModel instead of Pydantic base model (I think what was happening was that I didnt define the config and set validate_assignment to True)